tests/test_container.py

```diff
@@ -11,11 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import requests
+
+from unittest.mock import NonCallableMagicMock, MagicMock, patch, call
```

Comment:
Nitpick... this sintax is good according to PEP8, but it's not diff friendly. I'd suggest not to group the imports. No big deal though.